src/bundle/DependencyInjection/Configuration/Parser/FieldType/RichText.php

```diff
@@ -193,11 +193,19 @@ public function addFieldTypeSemanticConfig(NodeBuilder $nodeBuilder)
 
         // RichText Toolbars configuration (defines list of Toolbars and Buttons enabled for current SiteAccess scope)
         $nodeBuilder
-            ->arrayNode(self::TOOLBARS_NODE_KEY)
-                ->useAttributeAsKey('name')
-                ->info('List of Toolbars and Buttons enabled for current SiteAccess scope.')
+            ->arrayNode(self::TOOLBAR_NODE_KEY)
```

Comment:
This BC break needs to be mentioned in upgrade notes @ezsystems/documentation-team